webrtc-stats.html

```diff
@@ -1791,7 +1791,8 @@ <h2>
               <dd>
                 <p>
                   Represents the total number of <a>RTCP RR</a> blocks received for this <a>SSRC</a> that contain
-                  a valid round trip time. This counter will not increment if the {{roundTripTime}} is undefined.
+                  a valid round trip time. This counter will not increment if the {{roundTripTime}} is can not
+                  be calculated because no <a>RTCP Receiver Report</a> with a DLSR value other than has been received.
```

Comment:
Would it be clearer to say something along the lines of: "This counter will remain 0 until a <a>RTCP Receiver Report</a> with a non-zero DSLR has been received."?

Reply:
Agree that this whole things needs a bit of clarification, it tries to explain how RTCP RTT calculation works but does so in each definition instead of summarzing the RFC 3550 appendix in a single note.

(but that is for a separate change)